core/runtime/jsruntime.rs

```diff
@@ -1178,7 +1178,7 @@ impl JsRuntime {
       //     skip_op_registration: true
       //   }
       // ) {
-      if init_mode == InitMode::New {
+      if init_mode.needs_ops_bindings() {
```

Comment:
I don't think this will work sadly - this will incur additional overhead on each startup because the bindings that are in the snapshot will be overwritten with new bindings each time.

Reply:
hmm i was hoping it would be small enough since it only affects deno test/bench/jupyter, but it does look like it slows down `deno test` startup by about 10% which isn't great.

maybe we could split extensions between ones in the snapshot and ones not in the snapshot, then set up a new virtual ops module with _only_ the ones not in the snapshot?

Like `ext:core/ops` would be untouched, but for extensions not in the snapshot we would create a new `ext:core/extra-ops` with those ops?

Not sure it's much better than `Deno.core.ops` though, hmm

Reply:
I think `ext:core/extra-ops` would work, but that would require us to keep track which ops are already in the snapshot and which were added as "extras" (maybe we already do that, I don't really remember 😬). 

I wish we could modify synthethic module (or override it). Maybe @devsnek has any idea how that would be possible.